src/Simplex/Chat/Messages.hs

```diff
@@ -676,8 +678,10 @@ data CIStatus (d :: MsgDirection) where
   CISSndNew :: CIStatus 'MDSnd
   CISSndSent :: SndCIStatusProgress -> CIStatus 'MDSnd
   CISSndRcvd :: MsgReceiptStatus -> SndCIStatusProgress -> CIStatus 'MDSnd
-  CISSndErrorAuth :: CIStatus 'MDSnd
-  CISSndError :: String -> CIStatus 'MDSnd
+  CISSndErrorAuth :: CIStatus 'MDSnd -- deprecated
+  CISSndError :: String -> CIStatus 'MDSnd -- deprecated
+  CISSndDeliveryError :: SndDeliveryError -> CIStatus 'MDSnd
+  CISSndDeliveryWarning :: SndDeliveryWarning -> CIStatus 'MDSnd
```

Comment:
```suggestion
  CISSndError :: SndError -> CIStatus 'MDSnd -- deprecated
  CISSndWarning :: SndWarning -> CIStatus 'MDSnd
```

Reply:
I don't think there is a need to remove or deprecate AUTH error, it is kind of special.

Also CISSndError can be extended so that if it fails to parse it'll be put to other.